src/sentry/api/endpoints/sentry_internal_app_tokens.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+from rest_framework import status
+
+from sentry.api.bases import (
+    SentryInternalAppTokenPermission, SentryAppBaseEndpoint,
+)
+from sentry.models import SentryAppInstallation, SentryAppInstallationToken
+from sentry.features.helpers import requires_feature
+from sentry.mediators.sentry_app_installation_tokens import Creator
+from sentry.constants import INTERNAL_INTEGRATION_TOKEN_COUNT_MAX
+from sentry.api.serializers.models.apitoken import ApiTokenSerializer
+
+
+class SentryInternalAppTokensEndpoint(SentryAppBaseEndpoint):
+    permission_classes = (SentryInternalAppTokenPermission, )
+
+    @requires_feature('organizations:sentry-apps', any_org=True)
+    def post(self, request, sentry_app):
+        if not sentry_app.is_internal:
+            return Response('This route is limited to internal integrations only',
+                            status=status.HTTP_403_FORBIDDEN
+                            )
+
+        sentry_app_installation = SentryAppInstallation.objects.get(sentry_app=sentry_app)
+        curr_count = len(SentryAppInstallationToken.objects.filter(
+            sentry_app_installation=sentry_app_installation))
```

Comment:
We should use

```python
SentryAppInstallationToken.objects.filter(sentry_app_installation=sentry_app_installation).count()
```

so that it makes a `SELECT count(*)` query instead of pulling all the columns.